packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -673,6 +686,36 @@ export enum ScalingProcess {
   ADD_TO_LOAD_BALANCER = 'AddToLoadBalancer'
 }
 
+
+/**
+ * Health check settings
+ */
+export interface HealthCheckConfiguration {
+  /**
+   * Specifies the service to use for the health checks
+   *
+   * If you configure an Auto Scaling group to use ELB health checks, it considers the instance unhealthy
+   * if it fails either the EC2 status checks or the load balancer health checks.
+   *
+   * @default HealthCheckType.EC2
+   */
+  readonly type?: HealthCheckType;
+
+  /**
+   * Specified the time Auto Scaling waits before checking the health status of an EC2 instance that has come into service
+   *
+   * This property required with ELB health check type.
+   *
+   * @default -
```

Comment:
I couldn't find a current default value in the CloudFormation documentation. The [API Reference](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_CreateAutoScalingGroup.html) mentions a default of 0. Should I use `Duration.seconds(0)` in the JSDoc?